src/NuGetGallery/NuGetGallery.csproj

```
@@ -1864,6 +1865,7 @@
     <Content Include="Views\Authentication\Register.cshtml" />
     <Content Include="Views\Packages\ContactOwners.cshtml" />
     <Content Include="Views\Packages\DisplayPackage.cshtml" />
+	<Content Include="Views\Packages\DisplayPackageV2.cshtml" />
```

It looks like the spacing is off here - there's a tab instead of 4 spaces:

```suggestion
    <Content Include="Views\Packages\DisplayPackageV2.cshtml" />
```